ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -798,7 +798,7 @@ && shouldBootstrapDumpAcidTable(table.getTableName())) {
     return !ReplUtils.tableIncludedInReplScope(work.oldReplScope, table.getTableName());
   }
 
-  private boolean isTableSatifiesConfig(Table table) {
+  private boolean isTableSatisfiesConfig(Table table) {
```

Comment:
if we're about grammar, and this is a private method (we won't break anything), isn't it "doesTableSatisfyConfig instead?